Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,96 @@
+Determine the compatible supported OS to use as host to build. Choose the host with same processor architecture as that of the new targeted platform.  
+
+**Stage 1:**
+Build bootstrapping .NET SDK and toolset.
+- prep.sh (args???)  
+ Downloads platform-native version of .NET SDK and toolset. So if you are building on x64 you get x64.
+- build.sh (args???)  
+Create source built .NET SDK which will be utilized to bootstrap in stage 2.
+
+**Stage 2:**
+Use source built .NET SDK and toolset created in stage 1 to build .NET SDK from source. No need to run prep.sh in this stage.
+- Build.sh (Arg?????)
+[UPDATE THE RID GRAPH???]
+
+**If no compatible OS/platform found**  
+In stage 1, create portable SDK (this step works for .NET 6, but not for .NET 7). [Define the steps in here???]
+
+## Building for New Architecture (Supported, RID Existing)
+**Supported Archs**  
+Officially building and testing for the following architectures - x64, Arm64  
+Community building for following architectures - x64, Arm64, S390, Arm32, IBM Z
+
+**Supported RIDs**  
+You can see the list of supported RIDs and the RID graph in the [runtime.json](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json). Learn more about RID catalog in [here](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids).
+
+To build for new supported platform, follow this two stage build process:  
+
+**Stage 1:**
+Build bootstrapping .NET SDK and toolset.
+- prep.sh (args???)  
+ Downloads platform-native version of .NET SDK and toolset. So if you are building on x64 you get x64. Build .NET on the targeted platform to get intended platform-native version of .NET.
+- build.sh (args???)  
+Create source built .NET SDK and toolset.
+
+**Stage 2:**
+Use source built .NET SDK and toolset created in stage 1 to build .NET SDK from source. No need to run prep.sh in this stage.
+- Build.sh (Args?????)
+
+## Building for New Architecture (Unsupported)
+[Define the recommended process and add the steps in this section???]
+
+**RIDs**  
+The RID graph or runtime fallback graph is a list of RIDs that are compatible with each other. You can see the list of supported RIDs and the RID graph in the [runtime.json](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json). Learn more about RID catalog in (here)[https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids].
+
+Approach 1: Building for unsupported architectures require cross-compilaton on the supported  paltform. Determine the compatible host to build which provides cross-compilation toolchain.
```

Comment:
I would include some info why either one of these approaches could/should be used.

Reply:
Removing approach 2 as it is not validated to work properly. This will require testing before recommending.